src/components/app/details/appDetails/SourceInfo.tsx

```diff
@@ -170,6 +170,15 @@ export const SourceInfo = ({
                         </div>
                     </Tooltip>
                 )}
+                {appDetails?.resourceTree && ConfigSyncStatusButton && (
+                    <div className="pl-8">
+                        <ConfigSyncStatusButton
+                            areConfigurationsDrifted={appDetails.resourceTree.hasDrift}
+                            appId={appDetails?.appId}
+                            envId={envId}
+                        />
+                    </div>
+                )}
```

Comment:
or we can can not of isVirtual
```suggestion
                {appDetails?.resourceTree && !isIsolatedEnv && ConfigSyncStatusButton && (
                    <div className="pl-8">
                        <ConfigSyncStatusButton
                            areConfigurationsDrifted={appDetails.resourceTree.hasDrift}
                            appId={appDetails?.appId}
                            envId={envId}
                        />
                    </div>
                )}
```